src/frontend/types/tracks.ts

```diff
@@ -105,6 +105,24 @@ export interface Thumbnail extends Resource {
   video: Video['id'];
 }
 
+/** A SharedLiveMedia record as it exists on the backend. */
+export interface SharedLiveMedia extends Resource {
+  active_stamp: Nullable<number>;
+  filename: Nullable<string>;
+  is_ready_to_show: boolean;
+  nb_pages: Nullable<number>;
+  show_download: boolean;
+  title: Nullable<string>;
+  upload_state: uploadState;
+  urls: Nullable<{
+    media?: string;
+    pages: {
+      [key in number]: string;
+    };
```

Comment:
@kernicPanel @lunika wouldn't it be easer to have an ordered array here ?

Reply:
What do you call an ordered array ? Can you provide an example of what you want ?

Reply:
`pages: string[]` with pages[0] beeing the first page, pages[5] beeing the 6th page

Reply:
For now we will keep it like this and will see if it's a problem.